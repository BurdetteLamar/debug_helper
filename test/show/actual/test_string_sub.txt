---
DebugHelperTest::StringSub (message='test_string_sub'):
  to_s: Lorem ipsum
  size: 11
  encoding: !ruby/encoding UTF-8
  ascii_only?: true
  bytesize: 11
