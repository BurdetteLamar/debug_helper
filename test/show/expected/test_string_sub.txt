---
DebugHelperTest::StringSub (message='test_string_sub'):
  DebugHelperTest::StringSub#to_s: Lorem ipsum
  DebugHelperTest::StringSub#size: 11
  DebugHelperTest::StringSub#encoding: !ruby/encoding UTF-8
  DebugHelperTest::StringSub#ascii_only?: true
  DebugHelperTest::StringSub#bytesize: 11
